### PR TITLE
feat: version support

### LINKED ISSUE #19: add version command and fill with ldflags
We need to provide version information about the rekor-sidekick binary.

please feel free to assign it to me @nsmith5 🙋🏻‍♂️ happy to be doing it

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

Fixes #19 

cc: @nsmith5

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -7,6 +7,13 @@ builds:
   goarch:
   - amd64
   - arm64
+  ldflags:
+      - "-s -w"
```
Comment (nsmith5) [inline .goreleaser.yaml:11]: White space looks off. This is supposed to be a YAML list right?

File: cli/version.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+var (
+	commit = "none"
+	date   = "unknown"
+	tag    = "dev"
+)
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Display the version",
+	Long:  "Display version of kubectl-htpasswd",
```
Comment (nsmith5) [inline cli/version.go:17]: kubectl-htpasswd?

**Review by nsmith5 — CHANGES_REQUESTED** [review]

File: cli/version.go (hunk 2 above)
Comment (developer-guy) [reply cli/version.go]: it's my bad 🤦🏻‍♂️

File: .goreleaser.yaml (hunk 1 above)
Comment (developer-guy) [reply .goreleaser.yaml]: Fixed, thank you. ✌️

File: cli/version.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+var (
+	commit = "none"
+	date   = "unknown"
+	tag    = "dev"
+)
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Display the version",
+	Long:  "Display version of rekor-sidekick"
```
Comment (nsmith5) [inline cli/version.go:17]: This line is missing a final `,`

**Review by nsmith5 — CHANGES_REQUESTED** [review]

**Review by nsmith5 — APPROVED** [review]

**Pull request merged**